core/src/main/java/com/github/weisj/darklaf/DarkLaf.java

```diff
@@ -117,6 +122,24 @@ public UIDefaults getDefaults() {
         return super.getDefaults();
     }
 
+    private void patchMacOSFonts(final UIDefaults defaults) {
+        Set<Map.Entry<Object, Object>> entries = defaults.entrySet();
+        for (Map.Entry<Object, Object> entry : entries) {
```

Comment:
Does it make sense to inline `entries` variable?

Reply:
Yes it does 👍